docker/scripts/cpu-throttle.sh

```diff
@@ -0,0 +1,10 @@
+while true; do
+    if ps -ef | grep driver | grep use-mock-keychain > /dev/null
+    then
+        PROCESS_PID=$(ps -ef | grep driver | grep use-mock-keychain | awk '{print $2}')
```

Comment:
Can you share a little with some comments how this works? What process id:s are finding/searching?

Reply:
`ps -ef | grep driver | grep use-mock-keychain
PROCESS_PID=$(ps -ef | grep driver | grep use-mock-keychain | awk '{print $2}')` This finds the pid of the selenium driver instance to which we are going to limit the cpu. For every run pid varies, because we kill the process and start it again. For continuous throttling, this script will look for pid change and throttle it immediately.